packages/composer/build.sh

```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/composer/composer
+TERMUX_PKG_DESCRIPTION="Composer is a tool for dependency management in PHP."
+TERMUX_PKG_LICENSE=MIT
+TERMUX_PKG_VERSION=2
+TERMUX_PKG_DEPENDS="php"
+
+termux_step_make_install() {
+  if [ -f $TERMUX_PREFIX/composer ]; then
+  rm $PREFIX/bin/composer
```

Comment:
1. Scripts must use tabs.
2. No conditionals here. `rm -f /file/name` is enough.
3. Deleting file from `$TERMUX_PREFIX` is not necessary. **Delete the rm step**
4. Never use `$PREFIX` but only `$TERMUX_PREFIX`.